### PR TITLE
Trailing slash correction during bootstrap service add.

### PR DESCRIPTION
**Pull request by gaganhegde**
**What type of PR is this?**
> /kind bug

**What does this PR do / why we need it**:
The URL in bootstrap and service add command do not accept URL of the form https://github.com/org/repo-name/ although it is a valid url. The changes are to accept it with unit tests.

**Have you updated the necessary documentation?**
* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

Is an edge case does not require documentation changes.

**Which issue(s) this PR fixes**:
https://issues.redhat.com/browse/GITOPS-492

### CODE REVIEW

**Comment by wtam2018** [comment]
This is literaly a one-liner fix.
image

File: pkg/cmd/utility/utility_test.go [hunk 1]
```diff
@@ -24,11 +24,14 @@ func TestAddGitSuffix(t *testing.T) {
 		{"suffix for empty string", "", ""},
 		{"suffix already present", "https://github.com/test/org.git", "https://github.com/test/org.git"},
 		{"suffix with a different case", "https://github.com/test/org.GIT", "https://github.com/test/org.GIT"},
+		{"trailing slash absent", "https://github.com/test/org.git", "https://github.com/test/org.git"},
+		{"trailing slash present", "https://github.com/test/org/.git", "https://github.com/test/org.git"},
```
Comment (wtam2018) [inline pkg/cmd/utility/utility_test.go:28]: https://github.com/test/org/.git is a bad URL and should be rejected.

File: pkg/cmd/utility/utility.go [hunk 2]
```diff
@@ -22,8 +22,8 @@ func AddGitSuffixIfNecessary(url string) string {
 	if url == "" || strings.HasSuffix(strings.ToLower(url), ".git") {
 		return url
 	}
-	log.Italicf("Adding .git to %s", url)
-	return url + ".git"
+	notifyUsers(url)
```
Comment (bigkevmcd) [inline pkg/cmd/utility/utility.go:25]: I think we can improve this,

Let's merge the behaviour of `notifyUsers` back into this, but do something like...

```Go
trimmed := strings.TrimSuffix(url, "/")

if trimmed != url {
  log.Italicf(`Trimmed "/" from the end of %q", url)
}

return trimmed + ".git"

**Review by bigkevmcd — APPROVED** [review]
